### DISCUSSION TITLE
What recommended tolerance for matrix balancing?

### DISCUSSION THREAD

**Discussion by gdolsten** [open2c/cooler#283] [General]
Hi,
I am using cooler to balance my .cools. I was curious what tolerance you recommend for balancing. When I take the row-wise sum of interaction frequencies in my intra-chromosomal contact matrix, I observe a spread of values above/around 1.0, with a variance of .0036. Are these reasonable values?

**Comment by nvictus** [comment]
Did you run balacing in `cis-only` mode? The default balancing regime is done genome-wide, rather than intra-chromosomally, so the _genome-wide_ row sums should have a variance of ~1e-6. How this affects variability in cis can really depend on the nature of trans vs cis data, but your value seems reasonable. Note that you can run balancing in different modes and save multiple weight columns.

In some cases, if trans levels are very high with very low signal-to-noise (lots of ligation artifacts), it can contaminate balancing weights to an extent that becomes visibly noticeable. In those cases, it might make more sense to use intra-chromosomally balanced maps.
